config/locales/components.navbar.en.yml

```diff
@@ -0,0 +1,6 @@
+en:
+  components:
+    navbar:
+      login: "Login"
+      signup: "Signup"
+      logout: "Logout"
```

Comment:
No newline at end of file (éditeur à configurer ?)

Reply:
Oups j'ai changé d'ordinateur autant pour moi.